test/valid-data-struct.test.ts

```diff
@@ -8,6 +8,10 @@ describe("valid-data-struct", () => {
     it("object-literal-expression", assertValidSchema("object-literal-expression", "MyType"));
 
     it("literal-object-type", assertValidSchema("literal-object-type", "MyType"));
+    it(
+        "literal-object-type-with-computed-props",
+        assertValidSchema("literal-object-type-with-computed-props", "MyType")
```

Comment:
I'm not in love with this foldername / test name because it's not really testing an object literal, it's testing a Type, which happens to use the `TypeLiteralNodeParser` parser. Open to suggestions!

Reply:
It's not perfect but seems okay. You could rename to `computed-props`. Instead of an object, we could export multiple types in the test file. We would want to mirror that change in https://github.com/vega/ts-json-schema-generator/pull/746.